src/osd/OSD.cc

```diff
@@ -869,33 +869,34 @@ bool OSDService::_check_full(s_names type, ostream &ss) const
     // or if -1 then always return full
     if (injectfull > 0)
       --injectfull;
-    ss << "Injected " << get_full_state_name(type) << " OSD ("
-       << (injectfull < 0 ? "set" : std::to_string(injectfull)) << ")";
+    ldpp_dout(dpp, 10) << __func__ << " Injected " << get_full_state_name(type) << " OSD ("
+             << (injectfull < 0 ? "set" : std::to_string(injectfull)) << ")"
+             << dendl;
     return true;
   }
+  ldpp_dout(dpp, 10) << __func__ << " current usage is " << cur_ratio << dendl;
```

Comment:
Actually, this has caused the "current usage" message to always come out at log level 10 and above.  This was not the case unless a full condition has happened.

Reply:
@markhpc My only requested change then is to only do this dout when cur_state >= type